src/renderer/graph/Curve.tsx

```diff
@@ -90,20 +90,15 @@ const Curve = ({
 
   const d = useMemo(() => line(data), [data, line]);
 
-  useEffect(() => {
-    if (ref.current) {
-      d3.select(ref.current).transition().duration(500).attr('d', d);
```

Comment:
Mostly cuz it animates without this line of code already. This line of code adds a 500ms delay for the transition rather than animating immediately with filter changes. I prefer the immediate animation as it fits better with the idea of direct manipulation so I opted to remove this. Thoughts?

Reply:
part of me thinks its nicer with the animation. I would consider lowering the delay duration. @h39s @flyingpandaboy thoughts